Elevel/Application/Features/TopicCommands/GetTopicListQuery.cs

```diff
@@ -0,0 +1,69 @@
+﻿using AutoMapper;
+using AutoMapper.QueryableExtensions;
+using Elevel.Application.Interfaces;
+using Elevel.Application.Pagination;
+using Elevel.Domain.Enums;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Elevel.Application.Features.TopicCommands
+{
+    public class GetTopicListQuery
+    {
+        public class Request : PagedQueryBase, IRequest<Response>
+        {
+            public Guid Id { get; set; }
+        }
+
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var topic = _context.Topics.AsNoTracking()
+                    .Include(x => x.Id);
```

Comment:
you should not include it, include is using just for collections, not for flat properties